README.md

```diff
@@ -1,4 +1,7 @@
 # Software Engineering Team Design Project
+## Reports
+
+[Link to Sprint 1 Report](https://github.com/Mackbellemore/soen-390-team07/blob/develop/docs/sprint1_report.md)
```

Comment:
```suggestion
[Link to Sprint 1 Report](./docs/sprint1_report.md)
```

We can use relative link to make the link valid on any branch (blobs)

Reply:
ah ya my b